src/ui/profile_preferences.ui

```diff
@@ -42,7 +42,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
```

Comment:
:information_source: When you save the form/dialogue from the stand-alone Qt Designer utility or the version integrated into the Qt Creator IDE it remembers the current tab you are on - you need to always switch back to the first tab (or avoid this change getting into the end PR) to prevent this (change of the "currentIndex" away from `0`) from appearing...

As it happens I have put code into the `dlgProfilePrefences` constructor to prevent this from having any effect (it forces the current tab to the first one automatically) - but basically this change is not wanted... :grinning: